content/en/docs/KRE/user/50_krt_v2/_index.md

```diff
@@ -0,0 +1,108 @@
+
+## Reactive approach
+
+Nodes will publish their results onto their own subject, is up to other nodes whether to subscribe and process its messages or not. This allows for organic growth as nodes can be developed then hooked up to other nodes freely.
+
+### Multiple responses
+
+Nodes now also have the ability to publish several to none messages from the same request. So now the _handler_ contract should look like this:
+
+- Golang: `handler(ctx *kre.HandlerContext, data *any.Any) error`
+
+- Python: `async def default_handler(ctx, data):`
+
+Messages will be published using the function _SendOutput_ provided by the context.
+
+### Filter messages
+
+Nodes can choose to which subtopic from their own subject they publish to. At the same time, they can choose to take messages from any subtopic of another node. This allows users to filter out messages and reduce stress on nodes.
+
+Nodes can subscribe to one or more subtopics from another node and to various nodes at the same time. To help with filtering, users can set up which _handlers_ are to be executed depending on the request's origin. Take into consideration nodes can only discern the node from which the request proceeds, not the subtopic or topic. For setup guidance consult [Runner SDK]({{< relref "docs/KRE/user/60_kais_runner_sdk" >}}).
+
+## Exitpoint
+
+The exitpoint is a user defined node and can take up any name, however it has to be later assigned in the krt.yaml file to the exitpoint attribute. Users can only define one exitpoint as this is the final node to which the entrypoint will subscribe.
+
+An exitpoint is required in all workflows. As the exitpoint is the only node that will communicate back to the entrypoint, it is recommended to handle merge strategies in it. For example, if users wish to implement "early exit" and "early reply" strategies, they should subscribe the exitpoint to nodes emitting these types of messages.
+
+## KRT YAML file
+
+The KRT YAML file serves the same purpose as its predecessor. However, now nodes are declared and defined inside the workflows, an exitpoint must be described per workflow and a _v2_ tag must be added to the _krtVersion_ field.
+
+Also, users must define node's subscriptions inside each node, subscription names must match declared node's names. _'entrypoint'_ name is reserved to the entrypoint KRE provides, in order for a node to subscribe to the entrypoint just type the reserved name in its subscriptions.
+
+Learn about the fields in the [KRT YAML specs]({{< relref "docs/KRE/user/50_krt_v2/specs" >}}).
+
+Example below is directly taken from our krt v2 demo repo:
+
+```yaml
+version: classificator-v1
+krtVersion: v2 # new tagged krt version
+description: Demo email classificator for branching features.
+entrypoint:
+  proto: public_input.proto
+  image: konstellation/kre-entrypoint:latest
+config:
+  variables:
+workflows:
+  - name: classificator
+    entrypoint: Classificator
+    exitpoint: exitpoint # name must match node's
```

Comment:
This may be confusing as the node name is the same as the field name, I'd rename the node name to `exitpoint-node`, for example, for making the example more clear.